docs/userguide/expectations.md

```diff
@@ -14,66 +14,213 @@ Expectation is a set of the expected value(s), actual values(s) and the matcher(
 
 Matcher is defining the comparison operation to be performed on expected and actual values. 
```

Comment:
Do you think we can put the definition, matcher and example into a single table? I think it's going to be way shorter maybe even fit into a single screen